minutes/2024-09-24__Image-based-linux-summit.md

```diff
@@ -0,0 +1,315 @@
+# ﻿Image-Based Linux Summit Berlin 24th September 2024
+
+## Attendee's projects
+* systemd
+* mkosi
+* SUSE: MicroOS/Tumbleweed
+* Red Hat: [image-builder/osbuild](https://osbuild.org), [bootc](https://github.com/containers/bootc), systemd, systemd-boot
+* Microsoft: confidential containers, Flatcar, Azure Boost, Mariner/Azure Linux
+* Edgeless Systems: Constellation, [Contrast](https://github.com/edgelesssys/contrast) (confidential containers), [uplosi](https://github.com/edgelesssys/uplosi)
+* NixOS: systemd-{repart,stub,boot,cryptenroll}, tvix-store, [Lanzaboote](https://github.com/nix-community/lanzaboote), UKIs, [NixOS Bootspec](https://github.com/NixOS/rfcs/blob/master/rfcs/0125-bootspec.md), PE binary manipulation in Rust
+* Canonical: Ubuntu Core, snaps, foundations/bootloader/package management
+* System Transparency: UKI, UAPI specs, systemd-cryptenroll, remote attestation, TPM eventlog, mkosi
+* Freedesktop-sdk
+* GNOME OS
+* Pengutronix: OE/Yocto, nspawn, RAUC, verity, OPTEE
+* Arch Linux: sbctl/go-uefi, signstar, mkinitcpio, arch-install-scripts
+* [PostmarketOS](https://postmarketos.org/)
```

Comment:
Could we also add Kairos here? it's following the the Trusted Boot implementation defined by UAPI and systemd. I gave a presentation at the conference :) 

```suggestion
* [PostmarketOS](https://postmarketos.org/)
* [Kairos](https://kairos.io/)
```

Reply:
sorry I see this is for the summit, removing